source/core/TDCodecVia.cpp

```diff
@@ -2258,7 +2278,8 @@ void TDViaFormatter::FormatData(TypeRef type, void *pData)
         case kEncoding_DimInt:
             {
                 IntMax intValue = ReadIntFromMemory(type, pData);
-                FormatInt(type->BitEncoding(), intValue);
+                Boolean is64Bit = type->IsA(&TypeCommon::TypeInt64) || type->IsA(&TypeCommon::TypeUInt64);
```

Comment:
Maybe we should add isInteger64 as a helper in https://github.com/ni/VireoSDK/blob/master/source/core/TypeAndDataManager.cpp#L869

We now have several places we check for a 64bit integer. The two places in this pull request and here: https://github.com/ni/VireoSDK/blob/master/source/io/JavaScriptInvoke.cpp#L94